ocdaction/ocdaction/templates/dashboard/act/task_edit.html

```diff
@@ -29,8 +37,12 @@
                     {% render_field field placeholder=field.label class+="form-control" %}
                 </div>
             {% endfor %}
-
-            <button class="btn btn-primary pull-left col-xs-2 col-xs-offset-5" type="submit">Save</button>
+            <div class="col-xs-6">
+                <button class="btn btn-primary pull-right" type="submit">Save</button>
+            </div>
+            <div class="col-xs-6">
+                <a class="btn btn-primary pull-left" onclick="return checkCancel()" href="{% url 'task-list' %}">Cancel</a>
+            </div>
```

Comment:
All changes made, except for changing the `<a>` tag to `<button>`.  If I do that, the javascript pop up does not work correctly.